tools/bumbershoot/idpqonvert/idpQonvertEmbedder.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0"?>
+<tool id="idpqonvertEmbedder" name="idpEmbedder" version="0.1.0">
+    <description>Embed human/mouse gene metadata into IDPicker files</description>
+    <requirements>
+        <requirement type="package" version="3.0.8789">bumbershoot</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Job Failed" />
+        <regex match="\nError:.*\n" source="both" level="fatal" />
+    </stdio>
+    <command>
+<![CDATA[
+        #set $input_name = $input.display_name
+        cp '$input' output;
```

Comment:
Like idpAssemble, when idpQonvert operates on an idpDB input file, it changes the input rather than creating a new output. When idpQonvert operates on pepXML/mzidentML files, it creates idpDB files.